src/HydrationRepository.js

```diff
@@ -3,9 +3,9 @@ class HydrationRepository {
     this.hydroData = hydroData;
   }
 
- returnUserData(id) {
-   return this.hydroData.filter(hydroItem => hydroItem.userID === id)
- }
+  returnUserData(id) {
```

Comment:
The name of this function is odd to me and doesn't really describe what it is doing. I would rename it to something like `findHydroItem`.